usage/gpio/README.md

```diff
@@ -1,87 +1,63 @@
-# GPIO: Raspberry Pi Models A and B
+# GPIO
 
-### An introduction to GPIO and physical computing on the Raspberry Pi
+A powerful feature of the Raspberry Pi is the row of GPIO (general purpose input/output) pins along the top edge of the board. A 40-pin GPIO header is found on all current Raspberry Pi boards (unpopulated on Pi Zero and Pi Zero W). Prior to the Pi 1 Model B+ (2014), boards comprised a shorter 26-pin header.
 
-One powerful feature of the Raspberry Pi is the row of GPIO (general purpose input/output) pins along the edge of the board, next to the yellow video out socket.
+![GPIO pins](images/gpio-pins-pi2.jpg)
 
-![GPIO pins](images/gpio-pins.jpg)
+Any of the GPIO pins can be designated (in software) as an input or output pin and used for a wide range of purposes.
 
-These pins are a physical interface between the Pi and the outside world. At the simplest level, you can think of them as switches that you can turn on or off (input) or that the Pi can turn on or off (output). Seventeen of the 26 pins are GPIO pins; the others are power or ground pins.
+![GPIO layout](images/gpio-numbers-pi2.png)
 
-![GPIO layout](images/a-and-b-gpio-numbers.png)
+*Note that the numbering of the GPIO pins is not in numerical order.*
 
-## What are they for? What can I do with them?
+*Note that GPIO pins 0 and 1 are present on the board (physical pins 27 and 28) but are reserved for advanced use (see below).*
 
-You can program the pins to interact in amazing ways with the real world. Inputs don't have to come from a physical switch; it could be input from a sensor or a signal from another computer or device, for example. The output can also do anything, from turning on an LED to sending a signal or data to another device. If the Raspberry Pi is on a network, you can control devices that are attached to it from anywhere\*\* and those devices can send data back. Connectivity and control of physical devices over the internet is a powerful and exciting thing, and the Raspberry Pi is ideal for this. There are lots of brilliant examples of physical computing on [our blog](https://www.raspberrypi.org/blog/).
+## Voltages
 
-**Note**: Not **literally** anywhere, of course. You need things like access to the network, a network capable computing device, and electricity. Please do not write to us to point this out. :)
+Two 5V pins and two 3V3 pins are present on the board, as well as a number of ground pins (0V), which are unconfigurable. The remaning pins are all general purpose 3V3 pins, meaning outputs are set to 3V3 and inputs are 3V3 tolerant.
```

Comment:
Perhaps "The remaining 28 pins are all ..." ?

Reply:
I'll let copy editors decide :)